### PR TITLE
Fix typos in Events/function/variables/options names Estabilished -> Established

### PR DESCRIPTION
**Pull request by un1versal**
#### Changes proposed

~~\- Fixes various typos and capitalization across varied files~~
~~\- Fixes remainder of bolierplating typos and omissions~~
- Fixes typos in Events/function/variables names containing Estabilished -> Established

@staticfox if you feel like having a looksee ;)

### CODE REVIEW

File: data/man/kvirc.fr.1 [hunk 1]
```diff
@@ -64,14 +64,14 @@ Use this port for connection.
 This option requires also a server name.
 .TP 8
 .B [ircurl]
-An url in the following form:
+An URL in the following form:
```
Comment (staticfox) [inline data/man/kvirc.fr.1]: It's `a URL` not `an URL`

**Comment by staticfox** [comment]
I have reservations on changing the KVS event handler name since that may break existing scripts? (I rarely dive in to the inner mechanics of KVS). I'll look in to it in a bit to verify if it will or not, but someone can clarify in the mean time.

**Comment by un1versal** [comment]
> I have reservations on changing the KVS event handler name since that may break existing scripts? (I rarely dive in to the inner mechanics of KVS). I'll look in to it in a bit to verify if it will or not, but someone can clarify in the mean time.

yes, I thought of that also, but 4.2.0 -> 5.x.x will break much anyway so in a sense one more one less...
That said sod it, Im not that invested in this correction **but either do all or none**

**Comment by un1versal** [comment]
I split the other changes and merged them as they were straight forward fixes,

I left this one since its the one that is really needs clarification/investigation

**Comment by un1versal** [comment]
@TheReign for advice/thoughts because he does or knows ppl that do scripting and probably be affected if this breaks things.

**Comment by un1versal** [comment]
One more try, lets see if @OmegaPhil can provide some insight.

**Comment by OmegaPhil** [comment]
I'm for fixing obvious typos in code, and as you're saying earlier, a major version change allows for breakage.

None of my scripts reference anything involving 'estabilished', so you could say that is unfair for me to comment here, however it still seems that KVIrc scripting is not a 'big thing' yet, so its not like mIRC where you risk breaking countless hours of work.

So I would go ahead with this change.

**Comment by un1versal** [comment]
Thx for taking the time to reply. 

The reason for asking is even if your scripts dont use these, you are more familiar with this things and  that experience and knowledge in form of feedback or whatever, helps more than you might think.

> however it still seems that KVIrc scripting is not a 'big thing' yet

Lets hope this is closer rather than further away into the future.

**Comment by wodim** [comment]
My vote is no. Unnecessary breakage.

**Comment by HelLViS69** [comment]
Il 16/apr/2016 16:54, "un1versal" notifications@github.com ha scritto:

> Thx for taking the time to reply.
> 
> The reason for asking is even if your scripts dont use these, you are
> more familiar with this things and that experience and knowledge in form of
> feedback or whatever, helps more than you might think.
> 
> > however it still seems that KVIrc scripting is not a 'big thing' yet
> 
> Lets hope this is closer rather than further away into the future.

Maybe it's better to rename the function, and add a new function with the
old wrong name ad an alias, including in the doc that it's deprecated

HelLViS69

**Comment by un1versal** [comment]
The functions are internal and are not exposed to or matter users.

@HelLViS69 

the only one which matters scripting wise is **one single event** OnIRCConnectionEstablished

FYI the internal KVIrc documentation referenced the right name already for this event **(you added that documentation according to git blame back in 2008)**, so if anything, anyone used doc, it wouldn't have worked and this is in fact a fix for a buggy event name.

Reference https://github.com/kvirc/KVIrc/blob/7bc5592c3a556d6fd8e1738d203a4e5b87a3a4ba/src/kvirc/kvs/event/KviKvsEventTable.cpp#L593-L597

**Comment by wodim** [comment]
Ah, I thought it was the event name. You could fix this too: 

https://github.com/kvirc/KVIrc/blob/a589f834e10f8767cbfb7ac26485cfcd07ff2214/src/kvirc/kvs/KviKvsScript.cpp#L151

https://github.com/kvirc/KVIrc/blob/a589f834e10f8767cbfb7ac26485cfcd07ff2214/src/kvirc/kvs/KviKvsScript.cpp#L159

**Comment by HelLViS69** [comment]
Il 16/apr/2016 17:15, "un1versal" notifications@github.com ha scritto:

> The functions are internal and dont matter to users.
> 
> the only one which matters scripting wise is one single event
> 
> @HelLViS69
> FYI the internal KVIrc documentation referenced the right name already
> (you added that documentation according to git blame), so if anything,
> anyone used that, it wouldn't have worked and this is in fact a fix for a
> buggy event name.

Ok, so just grep those name and in case sed on all files..

HelLViS69

**Comment by un1versal** [comment]
> Ah, I thought it was the event name. You could fix this too:
> 
> https://github.com/kvirc/KVIrc/blob/a589f834e10f8767cbfb7ac26485cfcd07ff2214/src/kvirc/kvs/KviKvsScript.cpp#L151
> 
> https://github.com/kvirc/KVIrc/blob/a589f834e10f8767cbfb7ac26485cfcd07ff2214/src/kvirc/kvs/KviKvsScript.cpp#L159

thats a bug right there, fix incoming, thx

**Comment by un1versal** [comment]
> Ok, so just grep those name and in case sed on all files..

Im sure this is all of them except on .po files those get handled automatically on next Transifex sync

**Pull request merged**